### ISSUE TITLE
CAIA Review: COE Accessibility Fixes

### ISSUE THREAD

**Issue by Midge-dev** [department-of-veterans-affairs/va.gov-team#60823]
## Info
Refer to [Accessibility Review of COE experience from VA Benefits & Claims#60647](https://app.zenhub.com/workspaces/benefits-non-disability-645913c7d909c20011380ae8/issues/gh/department-of-veterans-affairs/va.gov-team/60647)

Site Content IA provided an updated review of accessibility for the COE experience. Below are the persistent items that should be fixed. 

## Acceptance Criteria
- [x] It was recommended that the following be reworded: `Please go to your profile page` link to include the purpose of said link. For example, `Go to your profile page to update your contact information` provides both a purpose and a destination.

- [x] Step 3 of 4: Your VA loan history should have a Veteran's loans written as a description list.

- [x] Split up Step 1 of 4: Your VA loan history into multiple steps since it currently has too many questions including: date fields, address fields, and yes/no fields.

**Comment by Midge-dev** [comment]
## Status Update: 

Currently 2 of 3 of the above mentioned suggestions from A11y have been implemented, however, they seem to be producing undesirable results in the COE workflow:
-"Add another" button renders only once at the beginning of the workflow
-SIP notice disappears from some sections
-Loan intent Yes/No is supposed to conditionally hide the loan intent question. Currently it shows the question statically 
-Raises potential accessibility concerns regarding how many steps there actually seem to be. Says 6 steps but feels like more
